presto-main/src/test/java/io/prestosql/sql/planner/assertions/PlanAssert.java

```diff
@@ -57,4 +57,30 @@ public static void assertPlan(Session session, Metadata metadata, StatsProvider
                     resolvedFormattedPlan));
         }
     }
+
+    public static void assertPlanDoesNotMatch(Session session, Metadata metadata, StatsCalculator statsCalculator, Plan actual, PlanMatchPattern pattern)
```

Comment:
Negative assertions tend to hide problems, when actual value deviates from the "non-expected" in an unexpected way.